src/Diagram.js

```diff
@@ -360,27 +390,33 @@ class Diagram extends React.PureComponent {
 
     if (this.props.enableZoom) {
       return (
-        <PanAndZoomContainer
-          handleScroll={this.handleScroll}
-          containerRef={this.containerRef}
-          renderPanAndZoomControls={this.props.renderPanAndZoomControls}
-          scroll={this.state.scroll}
-          contentSpan={{ x: extremeX, y: extremeY }}
-        >
-          {({ zoom }) => this.renderChildren(extremeX, extremeY, zoom)}
-        </PanAndZoomContainer>
+        <>
+          <div id="minimap-root" style={{ position: 'absolute', zIndex: 10 }} />
```

Comment:
make one function renderMiniMapRoot and use at both places

Reply:
done